includes/class-wcmp-frontend-settings.php

```diff
@@ -38,7 +38,9 @@ function __construct() {
          * @return int
          */
         public function is_only_recipient_enabled() {
-            return $this->settings['only_recipient_enabled'] ? 1 : 0;
+            if (isset($this->settings['only_recipient_enabled'])) {
+                return $this->settings['only_recipient_enabled'] ? 1 : 0;
+            }
```

Comment:
je moet hier ook iets returnen (bijvoorbeeld 0)

Reply:
die `? 1 : 0` doet al een isset. Die `if (isset($this->settings['only_recipient_enabled'])) {` is dus niet nodig.

Reply:
Zie ook de andere is_signature_enabled, is_morning_enabled, is_evening_enabled en is_monday_enabled

Reply:
Done!